README.md

```diff
@@ -1,2 +1,23 @@
 # Animal-Competition
-OOP assignment. Have animals fight each other.
+OOP assignment. This program is a game where two people, or one person and a computer, will create Animals and have them compete against one another
+
+
+## Game Progression
+(The words in bold have explanations down below)<br>
+The game progresses as follows:
+First, either player versus player or player versus computer must be chosen.
```

Comment:
Looks like you may want to used an ordered list for these lines, rather than a paragraph:
```suggestion
1. First, either player versus player or player versus computer must be chosen.
```